ext/webextension/src/css/mpwd.css

```diff
@@ -321,6 +323,25 @@ no-op {
     margin-top:0.2em;
 }
 
+.configitem > textarea#treat_as_same_site {
+    width: 11.5em;
+    height: 13em;
+    font-size: 0.7em;
+    margin-left: -11.5em;
+    margin-top: 2em;
+    white-space: nowrap;
+}
+
+/* FIXME: I don't understand how the left alignment of inputs works here, 
```

Comment:
Thats alright :) makes me remember I want to fix this cleanly with flexbox (which was very new back in 2015)